libres/lib/enkf/enkf_main_jobs.cpp

```diff
@@ -189,15 +189,13 @@ static void enkf_main_export_runpath_file(enkf_main_type *enkf_main,
             else
                 runpath = util_alloc_sprintf(runpath_fmt, iens_value);
 
-            runpath_list_add(runpath_list, iens_value, iter_value, runpath,
-                             basename);
-
-            free(basename);
-            free(runpath);
+            runpath_list.emplace_back(iens_value, iter_value,
+                                      std::string(std::move(runpath)),
```

Comment:
I guess this is the argument against using `explicit` constructors.

Reply:
Maybe, but we have the problem that jobname may be NULL. I am not sure whether that is only the case in tests though.